.circleci/config.yml

```diff
@@ -14,7 +14,6 @@ jobs:
     environment:
       # these environment variables will be passed to the docker container
       - CIBW_ENVIRONMENT: PIP_CONFIG_FILE=buildconfig/pip_config.ini PORTMIDI_INC_PORTTIME=1 SDL_VIDEODRIVER=dummy SDL_AUDIODRIVER=disk
-      - CIBW_BUILD: cp* pp*
```

Comment:
I don't understand why this change is here.

Reply:
That line was basically redundant because cibuildwheel default is to just do that